packages/components/src/whats-new/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Guide, GuidePage } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import wpcom from 'wpcom';
+import proxyRequest from 'wpcom-proxy-request';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const WhatsNewGuide = () => {
+	const [ isLoaded, setIsLoaded ] = useState( false );
+	const [ showGuide, setShowGuide ] = useState( true );
+	const [ whatsNewData, setWhatsNewData ] = useState();
+
+	// Load What's New list on first site load
+	useEffect( () => {
+		const proxiedWpcom = wpcom();
+		proxiedWpcom.request = proxyRequest;
+		proxiedWpcom.req
+			.get( { path: '/whats-new/list', apiNamespace: 'wpcom/v2' } )
+			.then( ( returnedList ) => {
+				setWhatsNewData( returnedList );
+				setIsLoaded( true );
+			} );
+	}, [] );
+
+	const toggleWhatsNew = () => setShowGuide( false );
+
+	return (
+		<>
+			{ isLoaded && showGuide && (
+				<Guide
+					className="whats-new"
+					contentLabel={ __( "What's New at WordPress.com", 'full-site-editing' ) }
+					finishButtonText={ __( 'Close', 'full-site-editing' ) }
+					onFinish={ toggleWhatsNew }
+				>
+					{ whatsNewData.announcements.map( ( page, index ) => (
```

Comment:
If we only need the `announcements` we might as well pass only that to `setWhatsNewData`? Or are we planning to add more data trees eventually?